typescript/create-onchain-agent/src/cli.ts

```diff
@@ -171,6 +171,7 @@ async function init() {
           message: (prev, { network }) => {
             const walletDescriptions: Record<WalletProviderChoice, string> = {
               CDP: "Uses Coinbase Developer Platform (CDP)'s managed wallet.",
+              SmartWallet: "Uses Coinbase Developer Platform (CDP)'s Smart Wallet.",
```

Comment:
opinion: What should the name be? This determined the name in the CLI.

Should it be:
*  "SmartWallet" (conventionally correct, since all the other options are the prefix of their associated WalletProvider)
* "Smart Wallet" (more grammatically correct)
* CDP Smart Wallet (best branding)